src/components/AppNavigationToggle/AppNavigationToggle.vue

```diff
@@ -79,8 +79,8 @@ export default {
 
 button.app-navigation-toggle {
 	position: absolute;
-	top: 0;
-	right: 0;
+	top: 8px;
```

Comment:
Somehow I feel those should be standardize or something :thinking:  :shrug:

Reply:
What do you mean? Variables?

Reply:
yeah, like I see you're also aligning the input and menu of the appnavigation in talk, I assume you're also using 8px? I think that should really be standardized :)